pkg/ovs/openflow/interfaces.go

```diff
@@ -204,6 +204,7 @@ type Action interface {
 	LoadARPOperation(value uint16) FlowBuilder
 	LoadToRegField(field *RegField, value uint32) FlowBuilder
 	LoadRegMark(mark *RegMark) FlowBuilder
+	LoadRegMarks(marks ...*RegMark) FlowBuilder
```

Comment:
What is the difference from calling LoadRegMark multiple times in client? I don't think it is necessary to implement this in binding..

Reply:
Calling LoadRegMark can simplify the code of flow-building, for example：

Without `LoadRegMarks`, sentence of flow-building has to be broken when a matching / loading reg mark needs an if condition.

```go
// podClassifierFlow generates the flow to mark the packets from a local Pod port.
func (f *featurePodConnectivity) podClassifierFlow(podOFPort uint32, isAntreaFlexibleIPAM bool) binding.Flow {
	flowBuilder := ClassifierTable.ofTable.BuildFlow(priorityLow).
		Cookie(f.cookieAllocator.Request(f.category).Raw()).
		MatchInPort(podOFPort).
		Action().LoadRegMark(FromLocalRegMark).
		Action().GotoStage(stageValidation)
	if isAntreaFlexibleIPAM {
		// This is used to mark the packets from a local Antrea IPAM Pod port.
		flowBuilder = flowBuilder.Action().LoadRegMark(AntreaFlexibleIPAMRegMark).
			Action().LoadRegMark(RewriteMACRegMark)
	}
	return flowBuilder.Done()
}
```

With `LoadRegMarks`, sentence of flow-building doesn't need to be broken. A list of matching / loading reg mark is generated first.

```go
// podClassifierFlow generates the flow to mark the packets from a local Pod port.
func (f *featurePodConnectivity) podClassifierFlow(podOFPort uint32, isAntreaFlexibleIPAM bool) binding.Flow {
	regMarksToLoad := []*binding.RegMark{FromLocalRegMark}
	if isAntreaFlexibleIPAM {
		regMarksToLoad = append(regMarksToLoad, AntreaFlexibleIPAMRegMark, RewriteMACRegMark)
	}
	return ClassifierTable.ofTable.BuildFlow(priorityLow).
		Cookie(f.cookieAllocator.Request(f.category).Raw()).
		MatchInPort(podOFPort).
		Action().LoadRegMarks(regMarksToLoad...).
		Action().GotoStage(stageValidation).
		Done()
}
```

Reply:
The change makes sense to me. Code looks cleaner. I agree with @wenyingd we could just expand the existing function, I can't think of any negative impact given the callers of single register won't be affected anyway (I think we don't have to append "s" for this case).

Reply:
Thanks, updated.